.vib/mongodb/vib-publish.json

```diff
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/mongodb"
         },
-        "runtime_parameters": "ImF1dGgiOgogICJyb290UGFzc3dvcmQiOiAiN2RNeExmY3F4TiIKICAicm9vdFVzZXIiOiAicm9vdCIKInNlcnZpY2UiOgogICJwb3J0cyI6CiAgICAibW9uZ29kYiI6IDgwCiAgInR5cGUiOiAiTG9hZEJhbGFuY2VyIgo=",
+        "runtime_parameters": "YXV0aDoKICByb290UGFzc3dvcmQ6IDdkTXhMZmNxeE4KICByb290VXNlcjogcm9vdAogIHVzZXJuYW1lczoKICAgIC0ga2F5bGUKICBkYXRhYmFzZXM6CiAgICAtIGF3ZXNvbWVfZGIKc2VydmljZToKICBwb3J0czoKICAgIG1vbmdvZGI6IDgwCiAgdHlwZTogTG9hZEJhbGFuY2VyCg==",
```

Comment:
I see you are using this username and password:
` rootPassword: 7dMxLfcqxN
  rootUser: root`
While I don't mind this at all, I am curious if it would be possible to use the standard user/ComplicatedPassword combination? 
Just for the sake of consistency.

Reply:
Good point! Just applied your feedback 😁